plugin-server/src/types.ts

```diff
@@ -756,13 +756,18 @@ export interface EventDefinitionType {
     created_at: string // DateTime
 }
 
+export type PropertyTypeFormat = 'unix_timestamp' | 'YYYY-MM-DD hh:mm:ss' | 'YYYY-MM-DD' // needs to sync with front end types
```

Comment:
Would it be too soon to optimise for extendability here?

```suggestion
type DateTimePropertyTypeFormat = 'unix_timestamp' | 'YYYY-MM-DD hh:mm:ss' | 'YYYY-MM-DD' // needs to sync with front end types
export type PropertyTypeFormat = DatePropertyTypeFormat
```